examples/Example13/example13.cu

```diff
@@ -275,12 +279,14 @@ void example13(int numParticles, double energy, int batch, const int *MCIndex_ho
     stats->inFlight[ParticleType::Gamma]    = 0;
 
     constexpr int MaxBlocks        = 1024;
-    constexpr int TransportThreads = 32;
+    constexpr int TransportThreads = 128;
     int transportBlocks;
 
     int inFlight;
     int loopingNo         = 0;
     int previousElectrons = -1, previousPositrons = -1;
+    int maxInFlight = 0, localMinInFlight = 0;
```

Comment:
You should maybe initialize both of them to `chunk` in the beginning, which is the initial number of particles in flight?